pylint/plugins/hass_enforce_type_hints.py

```diff
@@ -359,6 +367,69 @@ class TypeHintMatch:
     ],
 }
 
+_CLASS_MATCH: dict[str, list[ClassTypeHintMatch]] = {
+    "config_flow": [
+        ClassTypeHintMatch(
+            classes={
+                "AbstractOAuth2FlowHandler",
+                "ConfigFlow",
+                "SchemaConfigFlowHandler",
+            },
```

Comment:
With `node.ancestors()` it shouldn't be necessary to include `AbstractOAuth2FlowHandler` and `SchemaConfigFlowHandler` here. It will always match on `ConfigFlow`.

Reply:
Done